civihr_employee_portal/features/civihr_employee_portal_features/civihr_employee_portal_features.info

```diff
@@ -43,6 +43,7 @@ features[field_instance][] = node-hr_documents-field_attachment
 features[field_instance][] = node-hr_documents-field_download
 features[field_instance][] = node-hr_documents-field_resource_type
 features[field_instance][] = node-hr_documents-field_short_description
+features[filter][] = html_without_filters
```

Comment:
The filter name is confusing to me, what does it mean by `html_without_filters`? what does it do actually?

Reply:
`html_without_filters` outputs HTML code without any filter. The problem with the other filters like `Full HTML` or `Filtered HTML` is that they almost always add extra markup, extra `br` or `p` or transform a URL in a link, etc. This filter has disabled all filters and it is perfect to print HTML in views sections like footers or headers, but also useful in other situations (I'm adding this description in the technical details too).

Reply:
Ok, then please document this Filter under Technical Details of the PR. Any new components should be documented with details of why it is necessary, and how to use it.

Reply:
Sure. Done.